src/Symfony/Component/DependencyInjection/Loader/XmlFileLoader.php

```diff
@@ -550,6 +550,7 @@ private function getArgumentsAsPhp(\DOMElement $node, string $name, string $file
                     $arguments[$key] = $arg->nodeValue;
                     break;
                 case 'constant':
+                case 'enumeration':
```

Comment:
do we even need to add a new value ? Isn't `constant` enough ?

Reply:
Wasn't sure of this one. I would say it is better for code readability, but it works well with `constant`. Also, if `constant` is enough, is `php/enum` useless? We could use `php/const` as well.

Reply:
indeed, that make sense. So it would be a matter of adding support in dumpers

Reply:
I changed the code to only add support of enumerations by using `php/const` in Yaml and `constant` in XML, which seems fine to me. Hope I understood what you aksed!